server/app/auth/oidc/applicant/LoginGovClientProvider.java

```diff
@@ -99,7 +99,7 @@ public OidcClient get() {
 
```

Comment:
We set a state generator on line 86. This was originally duplicately set on 102.

Reply:
I see that setStateGenerator() is called on the config, but this seems to be called on the client.  How does the config and the client interact?

Reply:
Oh I see, the config is used in civiformOidcLogoutActionBuilder.java:106

Reply:
`LoginGovClientProvider` indirectly extends `OidcClientProvider`, which has the base `OidcClient get()` method.

That base method calls `getConfig()` to get the `OidcConfiguration`, which should drop down to the lowest implementation available, here.

Let me know if you disagree.